pilot/cmd/pilot-agent/main.go

```diff
@@ -240,6 +242,7 @@ var (
 				opts := make(map[string]string)
 				opts["PodName"] = os.Getenv("POD_NAME")
 				opts["PodNamespace"] = os.Getenv("POD_NAMESPACE")
+				opts["MixerSubjectAltName"] = envoy.GetMixerSAN(opts["PodNamespace"])
```

Comment:
The option ```MixedSubjectAltName``` is only set to ease the template rendering of ```envoy.yaml```. Otherwise you need to put a lot of logic into the template

Reply:
It's fine to add the option - my comment is that we need a better way to compute it, and not sure of "PodNamespace" 
can be used. The mixer san is based on the ns where mixer is running.

Reply:
As a note - this is only used for the sidecar in front of pilot, not for the general sidecar. The general problem here is that 
we need a way to figure out the mixer SAN and pass it to pilot itself - so it can generate the right config when mixer is not running in same namespace with pilot.

And we may split policy from telemetry in different namespaces for security reasons.